IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -59,73 +58,105 @@ def extract_study_info(session, screen_id):
     return details_
 
 
-def describe_screen(screen_id):
+def clean_channel_item(channel_item):
+    if bool(channel_item[channel_item.find("(") + 1 : channel_item.rfind(")")]):
```

Comment:
this line is difficult for me to understand. What is happening? You can add a comment to clarify.

Reply:
Added documentation for `clean_channel_item()`. The line searches for parentheses in the stain or target name. If they exist, then it removes them and whatever is inside (like an abbreviation for the channel).

For example: one of the targets in your study channels is `"(actin cytoskeleton, golgi, and plasma membrane (AGP))"`. This function will remove (AGP). I'll have to see if this makes the most sense after collecting metadata for all studies, but many of the channel targets that I've seen thus far don't use abbreviations for the target names. 

Added in 28f83df91f26ccce8ab9ddc0052a3d786ccf3299.